lib/modules/auphonic/admin.js

```diff
@@ -261,8 +261,9 @@ var PODLOVE = PODLOVE || {};
 
 			data.length_timestring = $("#_podlove_meta_duration").val();
 			data.output_basename = $("#_podlove_meta_slug").val();
-			data.metadata.title = $("#title").val();
+			data.metadata.title = $("#_podlove_meta_title").val() || $("#title").val();
```

Comment:
While this is my personally preferred behavior (and what I think makes most sense with the semantics of this field), I realize it would be a breaking change for some folks, so probably worth treading carefully here.

Reply:
Your solution feels "more correct" but yes I'll have to think if this is an acceptable change for a feature release and how to communicate it.